Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs

```diff
@@ -356,7 +367,7 @@ public string[] GetTrackedClientIDs()
     protected string[] GetTrackedClientIDs (int columnIndex)
     {
       if (HasEditControl (columnIndex))
-        return GetEditControl (columnIndex).GetTrackedClientIDs();
+        return GetEditControl (columnIndex)!.GetTrackedClientIDs();
```

Comment:
Assertion

Reply:
Is it really necessary here? There is no way to link `HasEditControl` and `GetEditControl` but its pretty clear here imo.

Reply:
With the Mandatory-remark (see further down) this becomes moot anyway. so, yeah, okay.